src/commonComponents/button/Button.js

```diff
@@ -0,0 +1,41 @@
+//Libraries
+import React from 'react';
+import PropTypes from 'prop-types';
+
+//Helpers
+import _noop from 'lodash/noop';
+
+//CSS
+import './button.css';
+
+function Button(props) {
+  const { label, onClick, className, id, action } = props;
+
+  function handleClick() {
+    return onClick(id, 1);
+  }
+
+  return (
+    <button className={`${className} ${action}`} onClick={handleClick}>
```

Comment:
Why do you have two classNames here? can't you keep the base styling here itself